src/Data/Witherable.hs

```diff
@@ -273,6 +274,20 @@ hashNub :: (Witherable t, Eq a, Hashable a) => t a -> t a
 hashNub = hashNubOf witherM
 {-# INLINE hashNub #-}
 
+-- | A default implementation for 'mapMaybe'.
+mapMaybeDefault :: (Foldable f, Alternative f) => (a -> Maybe b) -> f a -> f b
+mapMaybeDefault p = foldr (\x xs -> case p x of
+    Just a -> pure a <|> xs
+    _ -> xs) empty
+{-# INLINABLE mapMaybeDefault #-}
+
+newtype FilterableWrapper f a = FilterableWrapper {unWrapFilterable :: f a} 
```

Comment:
Minor: I think the convention is to `unwrap`, with the name reflecting the underlying structure (WrappedAlternative or something).  http://hackage.haskell.org/package/base-4.12.0.0/docs/src/Control.Applicative.html#WrappedMonad

Reply:
Fixed :)